### PR TITLE
[Merged by Bors] - chore: minimize some imports

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by mo271** [comment]
!bench

File: Mathlib/Analysis/Normed/Algebra/Exponential.lean [hunk 1]
```diff
@@ -70,7 +67,6 @@ to avoid collisions with the `Real` or `Complex` namespaces.
 
 As of 2023-11-16 due to bad instances in Mathlib
 ```
```
Comment (mo271) [inline Mathlib/Analysis/Normed/Algebra/Exponential.lean:69]:
```suggestion
```
import Mathlib
```

**Comment by mo271** [comment]
!bench

**Review by grunweg — APPROVED** [review]
All changes are either reordering imports to be more alphabetical, or removal.
Most of these are transitive, but I don't think removing these hurts, so LTGM. Thanks!
maintainer merge

**Comment by mo271** [comment]
@euprunin This is just some repeated calls of #min_imports. I wrote a some commands in the shell to automate it a bit.
Perhaps `!bench` will start working again at some point...

**Comment by semorrison** [comment]
Let's wait until `!bench` is back, but if that doesn't report anything scary please go ahead and merge. Thanks!

bors d+

**Comment by mo271** [comment]
!bench

**Comment by mo271** [comment]
!bench

**Comment by mo271** [comment]
rebased so `!bench` has something to compare against (hopefully)

**Comment by mo271** [comment]
!bench

**Pull request closed without merging**